lib/Doctrine/Common/Collections/AbstractLazyCollection.php

```diff
@@ -318,9 +318,12 @@ public function offsetGet($offset)
     }
 
     /**
-     * {@inheritDoc}
+     * @param TKey|null $offset
+     * @param T         $value
      *
      * @return void
+     *
+     * @psalm-suppress PossiblyNullArgument
```

Comment:
This is because:

```
ERROR: PossiblyNullArgument - lib/Doctrine/Common/Collections/AbstractLazyCollection.php:330:38 - Argument 1 of Doctrine\Common\Collections\Collection::offsetSet cannot be null, possibly null value provided (see https://psalm.dev/078)
        $this->collection->offsetSet($offset, $value);
```

Maybe `ArrayAccess` should explicitly say that `$offset` could be T|null, I'll try to take a look

Reply:
Please ignore this error via the psalm.xml.dist file and add a link to your Psalm issue as a comment.